### ISSUE TITLE
Bug Report: Moving notes to their original position makes them unselectable

### ISSUE THREAD

**Issue by amyspark-ng** [FunkinCrew/Funkin#3761]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Web/HTML5)

### Browser

None

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

After selecting a bunch of notes and unselecting i can't select them again, pretty weird

https://github.com/user-attachments/assets/d3e99e09-d54f-424c-8893-80bf9b3009db

**Comment by Hundrec** [comment]
Just played around with this a bit and found that it's something between a visual error and a serious inconvenience. 

If you select a group of notes, then drag and drop them, there's a chance they'll become unhighlighted. If you then scroll down and back up to the notes, they'll be selected again. This makes it seem like the notes are "ghost-selected", but that's not the case either. If you drag and drop the notes, then try to move one note as if the group were still selected, only the single note reacts.
You can also create a bunch of different weird interactions by scrolling to view only parts of the selected group as well.

So scrolling fixes unselectable notes, but it's enough of an inconvenience and causes enough confusion to warrant a PR from one of our coder friends (much appreciated!).

**Comment by Lasercar** [comment]
Yeah, the issue is that when you drag and drop, it moves them, but it doesn't also deselect them after moving.

**Comment by Lasercar** [comment]
This issue now has a pending PR. (Yay! I fixed it!)
